dataworks-aws-ucfs-stub.tf

```diff
@@ -0,0 +1,77 @@
+resource "github_repository" "dataworks-aws-ucfs-stub" {
+  name             = "dataworks-aws-ucfs-stub"
+  description      = "A repo for DataWorks AWS UCFS export server stub"
```

Comment:
is it just for the export server or potentially for any stubbed infrastructure?

Reply:
It's for the export server initially, but in an ideal world we'll move the stub VPC and Kafka broker in here as well.

Reply:
maybe this then:
A repo for DataWorks AWS UCFS stub infrastrucutre

Reply:
done